doc/api/async_hooks.md

```diff
@@ -3,9 +3,16 @@
 <!--introduced_in=v8.1.0-->
 
 > Stability: 1 - Experimental
+> state. Please migrate to other APIs.
 
 <!-- source_link=lib/async_hooks.js -->
 
+There are other APIs available to replace `async_hooks` at least for most use
+cases. Migrate/use them instead of using `async_hooks`. For example:
```

Comment:
How about this then? cc @Trott 

```suggestion
We strongly discourage the use of the `async_hooks` API.
Other APIs that can cover most of its use cases include:
```

Reply:
I'm fine with that text if it works for @Flarna.